final/recommendations/alternate-eval.tex

```diff
@@ -13,15 +14,3 @@ \subsection{Adding support for alternative evaluation strategies}
 \texttt{``IEvaluationStrategy''}. The language designer can register alternative
 evaluation strategies with the REPL by extending the Guice module of the backend
 and overriding the \texttt{bindEvalStrategies} method.
-
-Finally, the evaluation strategy that will be used to evaluate input terms can
```

Comment:
Ah I see:

> While this has been taken into account for the design of the
product (see \cref{sec:esv-extensions}), only a DynSem evaluation strategy has
actually been implemented.

That's the reference you meant, isn't it?